code/game/machinery/oxygen_pump.dm

```diff
@@ -14,6 +14,7 @@
 	var/mask_type = /obj/item/clothing/mask/breath/emergency
 	var/icon_state_open = "oxygen_tank_open"
 	var/icon_state_closed = "oxygen_tank"
+	var/icon_state_active
```

Comment:
3 extra vars for icon state makes me angry but since this is just to add sprites I'll consider it out of scope to tell you to fix it.